libs/frontend/util-builtin/src/lib/resolve-style.ts

```diff
@@ -86,7 +86,7 @@ export const resolveThemeVariables = (
         console.warn(`Variable not found: ${variableName}`)
       }
 
-      return value
+      return value?.trim()
```

Comment:
This is for #235.
It seems that values like ` ${color-text}` (there's a space at the beginning!) is also considered valid in Style Row. Those leading space(s) cause some of the functions in `pubstudio-builder/libs/frontend/feature-color-picker/src/lib/color-picker-util.ts` to produce the incorrect results.